server/server.js

```diff
@@ -10,7 +10,11 @@ app.use(bodyParser.json());
 
 app.use(function (req, res, next) {
   res.setHeader('Access-Control-Allow-Origin', '*');
+<<<<<<< HEAD
```

Comment:
This is a merge conflict that needs to be resolved.

(But don't worry about it, as this PR is unnecessary - see my comment on the previous issue)